docs/release-1.2/existing-rhdh.md

```diff
@@ -64,121 +91,156 @@ metadata:
 EOF
 ```
 The value of `.data.npmrc` points to https://npm.registry.redhat.com.
-For testing RC plugin versions, update to `cmVnaXN0cnk9aHR0cHM6Ly9ucG0uc3RhZ2UucmVnaXN0cnkucmVkaGF0LmNvbQo=` (points to https://npm.stage.registry.redhat.com and can be accessed internally). If there is a need to point to multiple registries, modify the content of the secret's data from:
-```
+For testing RC plugin versions, update to `cmVnaXN0cnk9aHR0cHM6Ly9ucG0uc3RhZ2UucmVnaXN0cnkucmVkaGF0LmNvbQo=` (which points to https://npm.stage.registry.redhat.com and can be accessed internally within Red Hat). If there is a need to point to multiple registries, modify the content of the secret's data from:
+```yaml
   stringData:
     .npmrc: |
       registry=https://npm.registry.redhat.com
 ```
 to
-```
+```yaml
   stringData:
     .npmrc: |
       @redhat:registry=https://npm.registry.redhat.com
       @<other-scope>:registry=<other-registry>
 ```
 
 ### dynamic-plugins ConfigMap
-This ConfigMap houses the configuration for enabling and configuring dynamic plugins. To incorporate the orchestrator plugins, append the following configuration to the **dynamic-plugins** ConfigMap:
+This ConfigMap houses the configuration for enabling and configuring dynamic plugins in RHDH. 
+
+> Note: Be sure to review [this section](#identify-latest-supported-plugin-versions) to determine the latest supported Orchestrator plugin `package:` and `integrity:` values, and update the dynamic-plugin ConfigMap entries accordingly. The samples in this document may not be the latest. 
```

Comment:
I'm not sure if anchors are rendered properly on our website, can you please check?
In addition, should we refer to a specif version (i.e. `1.2`)?

Reply:
> I'm not sure if anchors are rendered properly on our website, can you please check

Would this actually render in the website (parodos.dev site)? The flow I used to get here is:
- Access website https://www.parodos.dev/1.2/docs/installation/installation-on-existing-rhdh/ 
- Click link in *The installation steps are detailed [here](https://github.com/parodos-dev/orchestrator-helm-operator/blob/main/docs/release-1.2/existing-rhdh.md).*
- Which brings the user to github where this doc is rendered md.  The anchor works in github markdown at least

> In addition, should we refer to a specif version (i.e. 1.2)?

I purposely left the version out - the `identify-latest-supported-plugin-versions` section directs the user to reference the Orchestrator CRD installed by the particular Orchestrator Operator. The default plugin versions in the CRD are the supported plugin versions for the Operator release, so they should be used. 

Including @jordigilh and @jenniferubah  on using the CRD for this purpose.